Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'sinatra'
 group :test do
```

Comment:
Noticed in commits: broken iterations uploaded, uploads not as frequent as they could be.

Reply:
Very thorough reviewing, @JRMBowers!

Reply:
Nice work